packages/backend/src/peripherals/blockscout/BlockscoutClient.ts

```diff
@@ -0,0 +1,100 @@
+import { Logger } from '@l2beat/backend-tools'
```

Comment:
why are we creating a new one?

Reply:
did you forget to delete old one?

Reply:
I have created new on as the one from `shared` library seems to be working with different version of BS api. Will confirm with tooling team if it can be deleted.

Reply:
Iu uses different underlying API so renamed it to `BlockscoutV2Client` and moved to shared